src/dmd/backend/el.d

```diff
@@ -185,7 +185,7 @@ elem *el_var(Symbol *);
 elem *el_settype(elem *,type *);
 elem *el_typesize(type *);
 elem *el_ptr(Symbol *);
-void el_replace_sym(elem *e,Symbol *s1,Symbol *s2);
+//void el_replace_sym(elem *e,Symbol *s1,Symbol *s2);
```

Comment:
I don't think body-less function declarations should conflict, unless their signature is ambiguous.

Reply:
It's implemented in exactly the opposite way as DRuntime abused the non-existence of such a check. See the "hack"  below.